lib/calyx.rb

```diff
@@ -173,4 +176,42 @@ def generate
       registry[:start].evaluate
     end
   end
-end
+
```

Comment:
This whole section is probably a good candidate for breaking out into a separate `data_template.rb` file.